File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int cnt = 0; 
        int answer = 0;
        for(int i = 0; i<nums.length; i++){
            if(nums[i]==1)
                cnt++;
            else
                cnt=0;
            answer = cnt > answer ? cnt : answer;
        }
        
        return answer;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cnt = 0
        result = 0

        for i in nums:
            if i == 1:
                cnt+=1
            else:
                cnt = 0
            
            result = max(result, cnt)
        
        return result